src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,4 +1,4 @@
-import { IExecutionResult, IHttp, IRead } from '../accessors';
+import { IExecutionResult, IHttp, IModify, IRead } from '../accessors';
```

Comment:
and another thought: Implementation wise the non-support of interface in Javscript would boil down to checking whether a method is implemented in the particular class (or originally belongs to the object's prototype, depending on which layer you look at it). You could also provide a Rocketlet Superclass implementing all interfaces. By inheriting from that class, the JS-developer gets an idea of which are the possible methods (although it'd be quite a few of them) and the framework can check whether the method's been overridden.